KERNEL_CONNECTION.md

```diff
@@ -0,0 +1,128 @@
+# Custom kernel connection example (inside Docker)
+
+**Note**: Hydrogen now supports using kernel gateways ([see the instructions in the main page](https://github.com/nteract/hydrogen/blob/master/README.md)). Using that option might be simpler and will allow you to use the functionality in Windows or Mac very easily.
```

Comment:
This sounds to me a bit like kernel gateways aren't supported on Linux which isn't the case.